ui/app/AppLayouts/Communities/controls/ExtendedDropdownContent.qml

```diff
@@ -176,31 +185,24 @@ Item {
                             value: root.communityId
                         }
                     },
-                    ExpressionFilter {
-                        expression: {
-                            searcher.text
-
-                            if (model.shortName && model.shortName.toLowerCase()
-                                    .includes(searcher.text.toLowerCase()))
-                                return true
-
-                            return model.name.toLowerCase().includes(
-                                        searcher.text.toLowerCase())
+                    AnyOf {
+                        enabled: searcher.text !== ""
+                        SQUtils.SearchFilter {
+                            roleName: "shortName"
+                            searchPhrase: searcher.text
+                        }
+                        SQUtils.SearchFilter {
+                            roleName: "name"
+                            searchPhrase: searcher.text
                         }
                     }
                 ]
 
-                proxyRoles: ExpressionRole {
+                proxyRoles: FastExpressionRole {
                     name: "categoryLabel"
 
-                    function getCategoryLabelForType(category, type) {
```

Comment:
This was giving the endless warning spam, would eventually sometimes even freeze the app